### PR TITLE
Feature 30

### PR DESCRIPTION
**Pull request by jschmid1**

### CODE REVIEW

File: app/controllers/activities_controller.rb [hunk 1]
```diff
@@ -5,15 +5,34 @@ def index
...
+    @activity = Activity.new
+  end
 
+  def create
+    @activity = Activity.new(activity_params)
+
+    respond_to do |format|
+      if @activity.save
+        format.html { redirect_to @activity, notice: 'Activity was successfully created.' }
+        format.json { render action: 'show', status: :created, location: @activity }
```
Comment (kalabiyau) [inline app/controllers/activities_controller.rb]: we don't need html format now. More than that, I would recommend to switch to `respond_with`

File: app/models/activity.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 class Activity < ActiveRecord::Base
+  has_and_belongs_to_many :users
+  has_one :creator, class_name: "User"
```
Comment (kalabiyau) [inline app/models/activity.rb]: you can use class name rather than string representation of it

File: app/views/activities/index.html.slim [hunk 3]
```diff
@@ -3,3 +3,6 @@ h1 Servus
   ul
     li
       = a.name
+
+
+= link_to 'New', new_activity_path
```
Comment (kalabiyau) [inline app/views/activities/index.html.slim]: new line absent

File: app/views/activities/new.html.slim [hunk 4]
```diff
@@ -0,0 +1,14 @@
+
```
Comment (kalabiyau) [inline app/views/activities/new.html.slim]: excess line

File: app/views/activities/new.html.slim [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+= form_for @activity do |f|
+  fieldset
+    - if @activity.errors.any?
```
Comment (kalabiyau) [inline app/views/activities/new.html.slim]: I would go for generalized solution for errors in rails

Comment (kalabiyau) [inline app/views/activities/new.html.slim]: @jschmid1  that could be the idea http://asciicasts.com/episodes/211-validations-in-rails-3

File: app/views/activities/show.html.slim [hunk 6]
```diff
@@ -0,0 +1,5 @@
+Strong Name:
```
Comment (kalabiyau) [inline app/views/activities/show.html.slim]: html tags should be lowercased

File: app/views/activities/show.html.slim [hunk 7]
```diff
@@ -0,0 +1,5 @@
...
+= @activity.name
+
+div align="left"
```
Comment (kalabiyau) [inline app/views/activities/show.html.slim]: align is a style attribute, hence should only live in css

File: spec/features/activity_creation_feature_spec.rb [hunk 8]
```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+describe 'New Activites Page' do
+  feature 'List Activites' do
```
Comment (kalabiyau) [inline spec/features/activity_creation_feature_spec.rb]: please use empty lines to denote describe from it blocks

**Comment by kalabiyau** [comment]
@jschmid1 nicely done! let's look into the issues and fix them

**Comment by jschmid1** [comment]
And this is why the test fails

File: spec/features/activity_creation_feature_spec.rb [hunk 9]
```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
```
Comment (kalabiyau) [inline spec/features/activity_creation_feature_spec.rb]: please move that file to main activities feature

**Comment by kalabiyau** [comment]
@jschmid1 https://gist.github.com/kalabiyau/eb4950990359579318d8 that is fixed version. There was a `end` block location in the wrong place

**Comment by jschmid1** [comment]
test are passing now

**Comment by kalabiyau** [comment]
One more rebase

**Pull request closed without merging**